src/lib/jsonParseV2/parseContextAttributeCompoundValue.cpp

```diff
@@ -325,3 +325,144 @@ std::string parseContextAttributeCompoundValue
 
   return "OK";
 }
+
+
+
+/* ****************************************************************************
+*
+* parseContextAttributeCompoundValueStandAlone - 
+*/
+std::string parseContextAttributeCompoundValueStandAlone
+(
+  Document&            document,
+  ContextAttribute*    caP,
+  orion::ValueType     valueType
+)
+{
+  caP->compoundValueP            = new orion::CompoundValueNode();
+  caP->compoundValueP->name      = "TOP";
+  caP->compoundValueP->container = caP->compoundValueP;
+  caP->compoundValueP->valueType = caP->valueType;  // Convert to other type?
+  caP->compoundValueP->path      = "/";
+  caP->compoundValueP->rootP     = caP->compoundValueP;
+  caP->compoundValueP->level     = 0;
+  caP->compoundValueP->siblingNo = 0;
+
+  orion::CompoundValueNode*   parent  = caP->compoundValueP;
+
+
+  //
+  // Children of the node
+  //
+  if (caP->valueType == orion::ValueTypeVector)
+  {
+    int counter  = 0;
+
+    for (Value::ConstValueIterator iter = document.Begin(); iter != document.End(); ++iter)
+    {
+      std::string                nodeType  = jsonParseTypeNames[iter->GetType()];
+      orion::CompoundValueNode*  cvnP      = new orion::CompoundValueNode();
+      char                       itemNo[4];
+
+      snprintf(itemNo, sizeof(itemNo), "%03d", counter);
+
+      cvnP->valueType  = stringToCompoundType(nodeType);
+      cvnP->container  = parent;
+      cvnP->rootP      = parent->rootP;
+      cvnP->level      = parent->level + 1;
+      cvnP->siblingNo  = counter;
+      cvnP->path       = parent->path + "[" + itemNo + "]";
+
+
+      if (nodeType == "String")
+      {
+        cvnP->stringValue       = iter->GetString();
```

Comment:
Oh yes ...
All 'cvnP->xxxValue =' were supposed to be horizontally aligned. Don't know what happened to this one ...

But, now I know what happened to the 'previous one' (nextcoming - I'm reading the PR comment starting from the end) - cut 'n paste ...